spec/services/api/v1.0/catalog/share_info_spec.rb

```diff
@@ -33,13 +33,9 @@
     end
   end
 
-  context "when all group uuids exist" do
-    it_behaves_like "#process"
-  end
-
```

Comment:
Was this removal an accident or did it get removed in a rebase or something?

Reply:
I wonder if that was floating around from a rebase earlier.  I see it's a service but not versioned. So it shouldn't be there anyway.